language-server/modules/langserver-core/src/test/resources/definition/sources/defThread.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/io;
```

Comment:
We don't import standard library modules in lang tests. Shall we remove this?

Reply:
Removed.
Commit - 424e179fe8c78c1ca2d7cbc86c540901c4309843